platform/ui/src/components/tableList/TableList.js

```diff
@@ -9,14 +9,21 @@ export class TableList extends Component {
     defaultItems: PropTypes.object,
     children: PropTypes.node.isRequired,
     headerTitle: PropTypes.string,
+    headless: PropTypes.bool,
+  };
+
+  static defaultProps = {
+    headless: false,
```

Comment:
I'm a fan of improving the default UI components so we don't fork every little thing, you fine with this change to @ohif/ui @dannyrb?